platform/android/export/export.cpp

```diff
@@ -1339,8 +1345,8 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return str;
 		}
 	}
-	void _fix_resources(const Ref<EditorExportPreset> &p_preset, Vector<uint8_t> &p_manifest) {
 
+	void _fix_resources(const Ref<EditorExportPreset> &p_preset, Vector<uint8_t> &p_manifest) {
```

Comment:
It's not a bug in this PR, but this should be `r_manifest` to follow our coding convention.
```suggestion
	void _fix_resources(const Ref<EditorExportPreset> &p_preset, Vector<uint8_t> &r_manifest) {
```

Reply:
I can fix it in this PR, no big deal since it's just a refactoring